### PR TITLE
Rearranged TestUtils in indexing-solr to support shared test methods

### PR DESCRIPTION
**Pull request by karenhanson**
- Moved TestUtils.java out of solr folder to be shared among all indexing-solr test classes.
- Moved IndexingConsumerIT.populateTriplestore to TestUtils
- Took some methods from ConsumerTestUtil and moved to TestUtils

### CODE REVIEW

File: indexing-solr/src/test/java/info/rmapproject/indexing/solr/TestResourceManager.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import java.util.Set;
 
 import static info.rmapproject.indexing.IndexUtils.iae;
+import info.rmapproject.indexing.TestUtils;
```
Comment (emetsger) [inline indexing-solr/src/test/java/info/rmapproject/indexing/solr/TestResourceManager.java]: This import is unused

File: integration/src/test/java/info/rmapproject/indexing/kafka/IndexingConsumerIT.java [hunk 2]
```diff
@@ -1,42 +1,32 @@
 package info.rmapproject.indexing.kafka;
 
 import static info.rmapproject.indexing.IndexUtils.EventDirection.TARGET;
+import static info.rmapproject.indexing.TestUtils.getRmapObjects;
```
Comment (emetsger) [inline integration/src/test/java/info/rmapproject/indexing/kafka/IndexingConsumerIT.java]: Is it possible to only make the necessary changes to import statements?  When looking at an IT and a bunch of changes to imports, I just worry about something being added or removed.  It's hard, when looking this diff, if that is the case.  So typically, if imports need to be changed, I wouldn't rely on the auto-formatting of the IDE (i.e. using "optimize imports"), but rather make only the necessary changes.

File: integration/src/test/java/info/rmapproject/indexing/kafka/SaveOffsetOnRebalanceIT.java [hunk 3]
```diff
@@ -1,9 +1,10 @@
 package info.rmapproject.indexing.kafka;
 
 import static info.rmapproject.indexing.IndexUtils.EventDirection.TARGET;
+import static info.rmapproject.indexing.TestUtils.createSystemAgent;
```
Comment (emetsger) [inline integration/src/test/java/info/rmapproject/indexing/kafka/SaveOffsetOnRebalanceIT.java]: Same comment here on imports as on the previous IT.  Generally, "optimize imports" produces quite a bit of noise in a diff, and makes it hard to see what exactly changed.

File: indexing-solr/src/test/java/info/rmapproject/indexing/solr/TestResourceManager.java (hunk 1 above)
Comment (karenhanson) [reply indexing-solr/src/test/java/info/rmapproject/indexing/solr/TestResourceManager.java]: I believe it is used... perhaps when TestUtils was in the same package it didn't have to be imported? Not sure, but there are references to it.

**Pull request merged**